lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -151,10 +151,49 @@ function makeComparison(name, actualResult, expectedResult) {
   };
 }
 
+/**
+ * Delete expectations that don't match environment criteria.
+ * @param {LocalConsole} localConsole
+ * @param {LH.Result} lhr
+ * @param {Smokehouse.ExpectedRunnerResult} expected
+ */
+function pruneExpectations(localConsole, lhr, expected) {
+  const userAgent = lhr.environment.hostUserAgent;
+  const userAgentMatch = /Chrome\/(\d+)/.exec(userAgent); // Chrome/85.0.4174.0
+  if (!userAgentMatch) throw new Error('Could not get chrome version.');
+  const actualChromeVersion = Number(userAgentMatch[1]);
+
+  /**
+   * @param {*} obj
+   */
+  function failsChromeVersionCheck(obj) {
+    if (!obj._minChromiumMilestone) return false;
+    return actualChromeVersion < obj._minChromiumMilestone;
+  }
+
+  /**
+   * @param {*} obj
+   */
+  function pruneNewerChromeExpectations(obj) {
+    for (const key of Object.keys(obj)) {
+      const value = obj[key];
+      if (!value || typeof value !== 'object') continue;
```

Comment:
both the google and chromium style guides don't allow a brace-less `if` statement if there's an `else` clause, which I think is a good rule. In this case it seems like it would be more readable, as well, since their all independent and the first one is already explicitly `continue`ing.